chainbase/src/main/java/org/tron/common/storage/leveldb/LevelDbDataSourceImpl.java

```diff
@@ -142,11 +142,23 @@ private void openDatabase(Options dbOptions) throws IOException {
     if (!Files.isSymbolicLink(dbPath.getParent())) {
       Files.createDirectories(dbPath.getParent());
     }
-    database = factory.open(dbPath.toFile(), dbOptions);
-    if (!this.getDBName().startsWith("checkpoint")) {
-      logger.info("DB {} open success with writeBufferSize {} M, cacheSize {} M, maxOpenFiles {}.",
-          this.getDBName(), dbOptions.writeBufferSize() / 1024 / 1024,
-          dbOptions.cacheSize() / 1024 / 1024, dbOptions.maxOpenFiles());
+    try {
+      database = factory.open(dbPath.toFile(), dbOptions);
+      if (!this.getDBName().startsWith("checkpoint")) {
+        logger
+            .info("DB {} open success with writeBufferSize {} M, cacheSize {} M, maxOpenFiles {}.",
+                this.getDBName(), dbOptions.writeBufferSize() / 1024 / 1024,
+                dbOptions.cacheSize() / 1024 / 1024, dbOptions.maxOpenFiles());
+      }
+    } catch (IOException e) {
+      if (e.getMessage().contains("Corruption:")) {
+        logger.error("Database {} corrupted, please delete database directory({}) " +

```

Comment:
please refer https://github.com/tronprotocol/java-tron/blob/develop/framework/src/main/java/org/tron/core/db/Manager.java#L572

Reply:
> please refer https://github.com/tronprotocol/java-tron/blob/develop/framework/src/main/java/org/tron/core/db/Manager.java#L572

@halibobo1205 Just print the same message as L572? Or just exit when any exception is caught?

Reply:
1. Exit when any db exception is caught
2. Give a tip just like  [L572](https://github.com/tronprotocol/java-tron/blob/develop/framework/src/main/java/org/tron/core/db/Manager.java#L572) when db is corrupted.